azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -192,6 +192,22 @@ public KubernetesContainerizedImpl(final Props azkProps,
       logger.error("Unable to read kube config file: {}", exception.getMessage());
       throw new ExecutorManagerException(exception);
     }
+    // Add all the job types that are readily available as part of azkaban base image.
+    this.addIncludedJobTypes();
+  }
+
+  /**
+   * Populate the included job types set with all the types that are readily available as part of
+   * azkaban base image.
+   */
+  private void addIncludedJobTypes() {
+    INCLUDED_JOB_TYPES.add("hadoopJava");
```

Comment:
By any chance this list can come from property with comma separated values and you prepare set out of it? If property is not set then go with default? Just to keep it generic for open source.

Reply:
Initially I thought about that, but all these job types are integral part of the azkaban code itself, so then why we externalize these job types as part of azkaban configs. Also the configs are cluster specific, so for all the cluster we need to have comma separated list. Hence, decided to keep the the list in the same codebase itself. Moreover, it also does not make sense to accept the list from outside as they these jobs are part of the Azkaban code itself. Also there is a possibility of getting incomplete list if we externalize or allow users to pass the list.

Reply:
Good point.